server/src/controllers/orderHistory/orderHistory-controllers.ts

```diff
@@ -0,0 +1,15 @@
+import { Request, Response } from "express";
+import { fetchOrderHistory } from "./utils/orderHistory-utils";
+
+export const getOrderHistory = async (req: Request, res: Response) => {
+  try {
+    // Usa a função utilitária para buscar os dados
+    const orderHistory = await fetchOrderHistory();
```

Comment:
não precisava dessa função fetchOrderHistory não, só apagar essa função e deixar assim:

```suggestion
     const { data, error } = await supabase
      .from("order-history")
      .select("*");
      
      if(error) throw error;
```